main.py

```diff
@@ -1,17 +1,50 @@
 import cv2
 import numpy as np
-
+from imutils.video import VideoStream
```

Comment:
No need to use different libs which can be done by already imported library ie `cv2`.